srcpkgs/oilshell/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'oilshell'
+pkgname=oilshell
+version=0.6.0
+revision=1
+wrksrc=oil-${version}
+build_style=configure
+configure_args="--prefix=/usr $(vopt_with readline)"
+makedepends="$(vopt_if readline readline-devel)"
+short_desc="Oil, a new unix shell"
+maintainer="Caleb JA <skywise@tutanota.de>"
+license="Apache-2.0"
+homepage="https://www.oilshell.org"
+distfiles="${homepage}/download/oil-${version}.tar.xz"
+checksum=141e3a5c1997a696d65747966978aaa38921d77e303aad9e77b4ab4aedab84b7
+register_shell="/usr/bin/osh"
+nocross="oilshell is nocross"
+nostrip=yes
+
+build_options="readline"
+build_options_default="readline"
+
+pre_build() {
+	# Remove the strip wrapper
+	rm -f "${XBPS_WRAPPERDIR}"/strip
```

Comment:
Is it really necessary to remove the wrapper twice?

Reply:
I was following what I saw in [grub](https://github.com/void-linux/void-packages/blob/master/srcpkgs/grub/template)'s template file, but possibly not. I'll try using only one at a time and see what happens.

Reply:
It looks like it needs the entry in `pre_build` and `nostrip=yes` to work. Without `pre_build` it fails to compile, and without `nostrip=yes` it fails to run after being installed.

Reply:
Alright then, just wondered if this was really necessary.